<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual Serial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnAbrirPuerto">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txbSerialPort">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txbEnvio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txbRecibido">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorrar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Puerto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Envio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Recibidos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCambiarL">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cambiar Linea</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnviarNum">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar Num</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
